### PR TITLE
Assignment one

### PR DESCRIPTION
**Pull request by connor1484**
Assignment one pull request

### CODE REVIEW

**Comment by connor1484** [comment]
Assignment one done

**Comment by smarks** [comment]
in your root directory put a file called .gitignore

files containing the patterns below are excluded from Git

.class
target/

!.mvn/wrapper/maven-wrapper.jar

Mac OS

.DS_Store

STS

.apt_generated
.classpath
.factorypath
.project
.settings
.springBeans

IntelliJ IDEA

.idea
*.iws
*.iml
*.ipr

NetBeans

nbproject/private/
build/
nbbuild/
dist/
nbdist/
.nb-gradle/

File: src/main/java/edu/connor/advancedjava/Person.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/*
+ * Person class that will return the name and age of the person and also say Hello
+ */
+
+package edu.connor.advancedjava;
+import java.time.LocalDate;
+import java.time.Year;
+import java.time.Period;
+
```
Comment (smarks) [inline src/main/java/edu/connor/advancedjava/Person.java]: javadoc

File: src/main/java/edu/connor/advancedjava/Student.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+package edu.connor.advancedjava;
+import java.time.LocalDate;
+import java.time.Year;
+
+
+public class Student {
+
+    public static void main(String[] args) {
+
+        String name = "Connor";                             //name of student
```
Comment (smarks) [inline src/main/java/edu/connor/advancedjava/Student.java]: when I comment code here are the rules I have for myself
only comment lines that are hard to understand / long / confusing or might do something unexpected. 
if I end up writing lots of comments I fix my code.

File: src/test/java/edu/connor/advancedjava/PersonTest.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    private Person student;         //student Person object
+    private int age;                //age of Student
+    private String name;            //name of Student
+
+    /**
+     * Create the Person object to test
+     *
+     * @param testName name of the test case
+     */
+    @Before
```
Comment (smarks) [inline src/test/java/edu/connor/advancedjava/PersonTest.java]: // double slash for single line comments 
/* 
* multiple comments 
*/

File: src/test/java/edu/connor/advancedjava/PersonTest.java [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        LocalDate dateOfBirth = LocalDate.of(1980, 1, 20);
+        student = new Person(name, dateOfBirth);
+    }
+
+    @Test
+    public void testAge() {
+
+        //testing result of expected age
+        age = student.getAge();
+        assertEquals(38, age);
```
Comment (smarks) [inline src/test/java/edu/connor/advancedjava/PersonTest.java]: always use the form of assert that takes a string argument first and use that string to describe the test

**Comment by smarks** [comment]
have you completed all the changes I requested ?

**Comment by connor1484** [comment]
oh sorry @smarks, I did not. I thought you said not to worry about them for the first assignment.  I made sure to do them for assignment 2.  I can update them after work today and push the changes up though if you want.

**Comment by connor1484** [comment]
Hi @smarks 
Sorry about the confusion. I've updated my code and deleted the extra files.  Let me know if there are any other issues.

Thanks,
Connor

**Pull request merged**